py3status/modules/google_calendar.py

```diff
@@ -25,6 +25,8 @@
         (default 1)
     cache_timeout: How often the module is refreshed in seconds
         (default 60)
+    calendar_id: The ID of the calendar to display.
+        (default: "primary")
```

Comment:
```suggestion
        (default "primary")
```
```python
  ==============================
  google_calendar
  ==============================
  `calendar_id` default not in docstring add (default 'primary')
```

Reply:
> ```python
>   ==============================
>   google_calendar
>   ==============================
>   `calendar_id` default not in docstring add (default 'primary')
> ```
Thank you for pointing it out.

Fixed now